core/src/main/res/values/strings.xml

```diff
@@ -331,9 +331,11 @@
 
     <!-- Preferences -->
     <string name="storage_pref">Storage</string>
+    <string name="storage_sum">Managing phone storage, backups, exports...</string>
     <string name="project_pref">Project</string>
     <string name="queue_label">Queue</string>
     <string name="integrations_label">Integrations</string>
+    <string name="integrations_sum">External integrations (gpodder.net)</string>
```

Comment:
External services? I think the description should not repeat the title here. Users probably don't know what gpodder.net is. Maybe synchronization?

Reply:
`External services`